.github/workflows/daily-slack-report.yml

```
@@ -1,11 +1,169 @@
+            echo "$1" | jq -r '
+              if (.data.result | length) == 0 then empty
+              else .data.result[] | "• " + (.metric.section // "unknown") + ": " + .value[1]
+              end
+            ' | sed '/^$/d'
+          }
+
+          SAVED_SECTIONS="$(format_sections "$J_SAVED_BY_SECTION" || true)"
+          FAIL_SECTIONS="$(format_sections "$J_FAIL_BY_SECTION" || true)"
+
+          FAIL_RATE_FMT="$(python3 - <<PY
+  v=float("$FAIL_RATE"); print(f"{v*100:.2f}%")
```

actionlint (1.7.10)

could not parse as YAML: could not find expected ':'

(syntax-check)